### ISSUE TITLE
 tareas 26/1/2017

### ISSUE THREAD

**Issue by Martinnqn** [lifex/GrageasMagicas#5]
- [x]  Elegir qué poner en la pantalla opciones.
- [x] Pensar si poner los Strings en String.xml ahora e ir agregando los nuevos String de forma continua o terminar todo y luego reformatear todo el código con los respectivos Strings.
- [ ] Agregar botón musica en pantalla principal.
- [x] Agregar botón Ranking en pantalla principal.
- [x] Diseñar la Base de datos.
- [x] Agregar el formulario de login y creación de cuentas.

**Comment by giuliano-marinelli** [comment]
Los strings se agregan en los archivos de la carpeta strings en assets (libgdx funciona asi).


porque puedo editar tu comentario jeje

**Comment by Martinnqn** [comment]
claro, pero despues hay que reemplazar el name del string en todos los lugares donde lo usas
PD: deja de editar mis comentarios wexd jaja
